typings.json

```diff
@@ -1,9 +1,11 @@
 {
   "ambientDependencies": {
     "chalk": "registry:dt/chalk#0.4.0+20160317120654",
+    "form-data": "registry:dt/form-data#0.0.0+20160316155526",
```

Comment:
When installing `request` it wanted to install `form-data`. It is on my radar to remove `request`...